README.md

```diff
@@ -1,221 +1,276 @@
-
-Shameless Plug: Using this standalone library is a great start, but if you want to go further and have your tags centrally managed instead of having them hard-coded on the page, Check out [Krux's SuperTag](https://www.krux.com/data-management-platform-solutions/super-tag-management/), developed by the same authors as this library.
+Remote scripts, especially ads, block the page from doing anything else while they load. They contribute a large % to load times which
+[affects your bottom line](http://devnet.kentico.com/Blogs/Thomas-Robbins/September-2012/How-loading-time-affects-your-bottom-line-Infograp.aspx).
+*Asynchronous* ads do not block the page and can be delivered after core
+content - [Async FTW](http://www.krux.com/blog/krux-engineers/synchronous-versus-asynchronous-tags-whats-the-big-deal/).
 
+Why is it so hard to deliver ads asynchronously? Because they may contain calls to `document.write`, which expects to be handled synchronously.
+**PostScribe lets you deliver a synchronous ad asynchronously without modifying the ad code**.
 
 ### Approach
 
-Other tag writing libraries exist (see [alternatives](#alternatives)), but PostScribe is novel in it's use of what we call DOM Proxies, a way to ensure that the content is written as close to the way the browser would natively write the content with `document.write`/`innerHTML`. Read: it behaves just like the browser would, without convoluted parsing or hacks.
+Other tag writing libraries exist (see [alternatives](#alternatives)), but PostScribe is novel in its use of what we call DOM Proxies, a way to
+ensure that the content is written as close to the way the browser would natively write the content with `document.write`/`innerHTML`. Read: it
+behaves just like the browser would, without convoluted parsing or hacks.
 
 For more information:
 
-* [Presentation at HTML5devconf](http://youtu.be/ClzeilKwX10) by the author, Derek Brans
+* [Presentation at HTML5devconf](http://youtu.be/ClzeilKwX10) by the original author, Derek Brans
 * [Interactive Demo](http://run.plnkr.co/plunks/e8MJAx/) with side by side comparisons of other tag writers
-* [Documentation](https://github.com/krux/postscribe/tree/master/doc)
-* Browse the [raw](https://github.com/krux/postscribe/blob/master/postscribe.js) or [annotated](http://krux.github.com/postscribe/doc/postscribe.html) source code.
-
 
 # Getting Started
 
-PostScribe overrides document.write. It is best and safest to use PostScribe after DOM is ready.
+PostScribe overrides `document.write`. It is best and safest to use PostScribe after DOM is ready.
+
+## Installation
+
+### Browser
+
+If you just want to use the script without installing anything, use the following to load the script from *cdnjs*:
+
+```html
+<script src="https://cdnjs.cloudflare.com/ajax/libs/postscribe/2.0.0/postscribe.min.js"></script>
+```
+
+### NPM
+
+You can include `postscribe` using *npm*:
+
+```console
+npm install --save-dev postscribe
```

Comment:
Postscribe seems more like a production dependency than a dev dependency, so it may be better to change this to just `--save`. Maybe add a note about Webpack/Browserify/JSPM/Rollup too, since postscribe is only meant to run in the browser.